common/crypto_utils/avalon_crypto_utils/signature.py

```diff
@@ -248,9 +245,19 @@ def generate_client_signature(
 
         encrypted_session_key_str = byte_array_to_hex_str(
             encrypted_session_key)
-        self.__encrypt_workorder_indata(
-            input_json_params, session_key,
-            session_iv, worker.encryption_key, data_key, data_iv)
+
+        logger.debug("Encrypting Workorder inData")
```

Comment:
We don't need to assume inData will be always present in work order request. We could have some workloads which does some fixed functionality without inData. You could add below check for inData similar to outData
````
 if "inData" in input_json_params and \
                len(input_json_params["inData"]) > 0:
```

Reply:
Done